src/test/java/ladder/domain/HorizontalLinesTest.java

```diff
@@ -0,0 +1,30 @@
+package ladder.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class HorizontalLinesTest {
+
+    @DisplayName("사다리 세로선 사이 가로선을 항상 번갈아가며 생성한다. (인덱스 기준 짝수 생성, 홀수 미생성)")
+    @ParameterizedTest
+    @ValueSource(ints = 5)
+    void connect(int lines) {
+        HorizontalLines horizontalLines = new HorizontalLines();
+        ConnectingStrategy connectingStrategy = new SequentialConnectionStratrgy();
+        horizontalLines.connect(lines, connectingStrategy);
+
+        List<Boolean> horizontalLineList = horizontalLines.getHorizontalLines();
+        for (int i = 0; i < horizontalLineList.size(); i++) {
+            if (i % 2 == 0) {
+                assertThat(horizontalLineList.get(i)).isTrue();
+                continue;
+            }
+            assertThat(horizontalLineList.get(i)).isFalse();
+        }
```

Comment:
```suggestion
        assertThat(horizontalLineList).containsExactly(true, false, true, false, true);
```
검증을 위해 추가적인 로직을 만드는 것 보단 하드코딩을 통해 원하는 값이 뭔지 명확하게 나타내는게 좋다고 생각해요 :)
어떻게 생각하시나요?